app/grandchallenge/invoices/tasks.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.db import transaction
+from django.utils.timezone import now
+
+from grandchallenge.core.celery import acks_late_micro_short_task
+from grandchallenge.invoices.emails import (
+    send_challenge_invoice_issued_notification,
+    send_challenge_invoice_overdue_reminder,
+)
+
+
+@acks_late_micro_short_task
```

Comment:
Maybe put it on the 2xlarge queue, just in case a lot of emails need to be sent.